### PR TITLE
Adding Projected Overloads vs Timeseries

### PR DESCRIPTION
**Pull request by jamestamplin**
added a section talking about the timeseries param.

@asciimike please tell me why it's all broken and wrong

This PR addresses issue https://github.com/covid-projections/covid-data-model/issues/< >

### Please Check
- [ ] Will the *current* schema in [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.md) be updated with this PR?
- [ ] Are tests passing?

### CODE REVIEW

File: api/README.V1.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ There are no rate limits
 Data is licensed under [Creative Commons 4.0 By Attribution](https://creativecommons.org/licenses/by/4.0/). You are welcome to share, copy, and redistribute it, as well as adapt it for your own works, we just ask that you provide attribution to the source (as we have done with [our data sources](https://github.com/covid-projections/covid-data-public#date-sources-for-current--future-use)).
 
 
-## Using the API
+## API Parameters
```
Comment (asciimike) [inline api/README.V1.md:20]: I preferred "Using the API"

**Comment by jamestamplin** [comment]
ack, feel free to revert

James Tamplin
tinyletter.com/jamest (2-3 emails / yr on what I'm up to)

File: api/README.V1.md (hunk 1 above)
Comment (jamestamplin) [reply api/README.V1.md:20]: OK, will revert.

**Comment by jamestamplin** [comment]
@asciimike - reverted to 'Using the API', please merge or tell me what else to fix :)

**Pull request merged**